serverless/pytorch/saic-vul/hrnet/nuclio/function-gpu.yaml

```diff
@@ -61,6 +61,8 @@ spec:
           value: pip3 uninstall torch torch vision -y
         - kind: RUN
           value: pip install torch==1.8.1+cu111 torchvision==0.9.1+cu111 torchaudio==0.8.1 -f https://download.pytorch.org/whl/torch_stable.html
+        - kind: RUN
+          value: pip install "numpy<1.24"
```

Comment:
Could you merge this with the `pip3 install -r requirements.txt` command, so that we don't needlessly download/install the most recent numpy version before downgrading it?

It would also be helpful if you could add a comment to explain why we're downgrading NumPy.

Reply:
> It would also be helpful if you could add a comment to explain why we're downgrading NumPy.

I opened SamsungLabs/ritm_interactive_segmentation#36, so you can just link to that.

Reply:
It looks like I push the commit at the same time of your comment :) Do you want me to change the comment to refer to this PR you opened or is it enough in the current format?

Reply:
Should be okay as-is.